Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -217,12 +217,24 @@ public async Task Push(Image x, string name, string? tag, string baseName, Actio
         manifestUploadContent.Headers.ContentType = new MediaTypeHeaderValue(DockerManifestV2);
         var putResponse = await client.PutAsync(new Uri(BaseUri, $"/v2/{name}/manifests/{x.GetDigest(x.manifest)}"), manifestUploadContent);
         string putresponsestr = await putResponse.Content.ReadAsStringAsync();
-        putResponse.EnsureSuccessStatusCode();
+
+        if (!putResponse.IsSuccessStatusCode)
+        {
+            string jsonResponse = await putResponse.Content.ReadAsStringAsync();
```

Comment:
this is recalculating the `putresponsestr` from line 219 - is putresponsestr used in other places? can either of these be removed?

Reply:
I didn't realize that, I assumed it would have to be recalculated after the "put". db85a7e is my first-pass workaround for that, assuming we don't actually need to recalc the contents based on the put.